Microsoft.DotNet.Interactive/Events/UnrecognizedCommand.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.DotNet.Interactive.Commands;
+
+namespace Microsoft.DotNet.Interactive.Events
+{
+    internal class UnrecognizedCommand : IKernelEvent
```

Comment:
`CommandNotRecognized` would be more in line with event naming conventions. Although I also think this is a developer error condition, so an exception or error channel would be more appropriate.